js/main.js

```diff
@@ -0,0 +1,12 @@
+var $form = document.querySelector('form');
+var $searchInput = document.querySelector('.search-input');
+
+function handleSubmit() {
+  event.preventDefault();
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://api.jikan.moe/v3/search/anime?q=' + $searchInput.value);
+  xhr.responseType = 'json';
+  xhr.send();
```

Comment:
You should add a `load` event listener so that you can check that you are getting data